portfolio/src/main/java/com/google/sps/data/Comment.java

```diff
@@ -0,0 +1,20 @@
+package com.google.sps.data;
+
+/** Class to store data associated with each comment left on page */
+public final class Comment {
+  private String name;
+  private String comment;
+
+  public Comment(String name, String comment) {
```

Comment:
I think "author" and "value" are clearer than "name" and "comment"

Reply:
Changed.